pom_parent.xml

```diff
@@ -63,27 +65,44 @@
             <version>${jakarta.json.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
I wonder if it's hard to have the depdencies needed to benchamrk hidden under a separate goal, e.g. something like 
```
mvn compile benchmark
```
Having benchmark as  mvn goal would help to  keep track on performance.

Also have benchmark sources and resources separated, e.g.
```
/src/benchmark/java
/src/benchmark/resources
```
or something like that. One of the main advantages of Titanium are min. dependencies.

Reply:
You're thinking of the guava dependency. That's only needed if we end up with using the bloom filter.

The benchmark dependencies are all scoped to `test`, which is the easiest way to not include them in the final build.

In RDF4J we've added a step to the maven process that generates the JMH jar files, which makes it possible to run the benchmarks from the command line. I usually just run them from IntelliJ using a JMH plugin. We can also add a main method to each benchmark that runs them, that's helpful for any developers using Netbeans or Eclipse.